zipline/src/hostTest/kotlin/app/cash/zipline/FlowTest.kt

```diff
@@ -241,4 +244,28 @@ internal class FlowTest {
     assertEquals(setOf(), endpointA.serviceNames)
     assertEquals(setOf(), endpointB.serviceNames)
   }
+
+  /** https://github.com/cashapp/zipline/issues/1039 */
```

Comment:
This test crashed on the old code attempting to use an Int serializer on a string.